includes/Classifai/Providers/OpenAI/ChatGPT.php

```diff
@@ -227,90 +227,108 @@ public function enqueue_editor_assets() {
+			&& 'openai_chatgpt' === $_GET['provider'] // phpcs:ignore
+		) {
+			wp_enqueue_script( 'jquery-ui-dialog' );
+			wp_enqueue_style( 'wp-jquery-ui-dialog' );
+
+			add_action(
+				'admin_footer',
+				static function () {
+					printf(
+						'<div id="js-classifai--delete-prompt-modal" style="display:none;"><p>%1$s</p></div>',
+						esc_html__( 'Are you sure you want to delete the prompt?', 'classifai' ),
+					);
+				}
+			);
 		}
 
-		$screen   = get_current_screen();
-		$settings = $this->get_settings();
-
-		// Load the assets for the classic editor.
-		if ( $screen && ! $screen->is_block_editor() ) {
-			if (
-				post_type_supports( $screen->post_type, 'title' ) &&
-				$this->is_feature_enabled( 'enable_titles' )
-			) {
-				wp_enqueue_style(
-					'classifai-generate-title-classic-css',
-					CLASSIFAI_PLUGIN_URL . 'dist/generate-title-classic.css',
-					[],
-					CLASSIFAI_PLUGIN_VERSION,
-					'all'
-				);
-
-				wp_enqueue_script(
-					'classifai-generate-title-classic-js',
-					CLASSIFAI_PLUGIN_URL . 'dist/generate-title-classic.js',
-					array_merge( get_asset_info( 'generate-title-classic', 'dependencies' ), array( 'wp-api' ) ),
-					get_asset_info( 'generate-title-classic', 'version' ),
-					true
-				);
+		// Load asset in new post and edit post screens.
+		if ( 'post.php' === $hook_suffix || 'post-new.php' === $hook_suffix ) {
+			$screen   = get_current_screen();
+
+			// Load the assets for the classic editor.
+			if ( $screen && ! $screen->is_block_editor() ) {
+				if (
+					post_type_supports( $screen->post_type, 'title' ) &&
+					$this->is_feature_enabled( 'enable_titles' )
+				) {
+					wp_enqueue_style(
+						'classifai-generate-title-classic-css',
+						CLASSIFAI_PLUGIN_URL . 'dist/generate-title-classic.css',
+						[],
+						CLASSIFAI_PLUGIN_VERSION,
+						'all'
+					);
+
+					wp_enqueue_script(
+						'classifai-generate-title-classic-js',
+						CLASSIFAI_PLUGIN_URL . 'dist/generate-title-classic.js',
+						array_merge( get_asset_info( 'generate-title-classic', 'dependencies' ), array( 'wp-api' ) ),
+						get_asset_info( 'generate-title-classic', 'version' ),
+						true
+					);
+
+					wp_add_inline_script(
+						'classifai-generate-title-classic-js',
+						sprintf(
+							'var classifaiChatGPTData = %s;',
+							wp_json_encode( $this->get_localised_vars() )
+						),
+						'before'
+					);
+				}
 
-				wp_add_inline_script(
-					'classifai-generate-title-classic-js',
-					sprintf(
-						'var classifaiChatGPTData = %s;',
-						wp_json_encode( $this->get_localised_vars() )
-					),
-					'before'
-				);
+				if (
+					post_type_supports( $screen->post_type, 'excerpt' ) &&
+					$this->is_feature_enabled( 'enable_excerpt' )
+				) {
+					wp_enqueue_style(
+						'classifai-generate-title-classic-css',
+						CLASSIFAI_PLUGIN_URL . 'dist/generate-title-classic.css',
+						[],
+						CLASSIFAI_PLUGIN_VERSION,
+						'all'
```

Comment:
Non-blocking question: Should `CLASSIFAI_PLUGIN_VERSION` be replaced with `get_asset_info` everywhere, including CSS?

Reply:
That's a good question. In looking through the codebase on all usages of `CLASSIFAI_PLUGIN_VERSION`, it seems we have a handful of places that still use this, mostly CSS files (though a couple JS files as well). I don't think there's any reason these need to use that constant anymore, they should all be able to use this (fairly) new `get_asset_info` function that pulls the version number from the generated asset file.

I don't think we need to block this PR based on that though, so I'm going to open a separate issue and we can fix that up in a new PR (and tag you on that once it's ready). Thanks!